docs/package.json

```diff
@@ -17,10 +17,9 @@
     "lint": "eslint . --ext=.tsx,.ts,.js,.md,.mdx --ignore-pattern 'node_modules/*' --ignore-pattern 'versioned_docs/*' --ignore-pattern 'versioned_sidebars/*' --fix",
     "lint-and-vale": "yarn run lint && yarn run vale",
     "generate-code-imports": "node scripts/generate-code-imports.js",
-    "sync-api-docs": "/bin/sh scripts/vercel-sync-api-docs.sh",
-    "build-api-docs": "tox -e sphinx-mdx && cp -rf sphinx/_build/mdx/sections/api/apidocs/* docs/api/python-api/",
-    "build-sphinx-object-inv": "tox -e sphinx && cp sphinx/_build/json/objects.inv static/.",
-    "rebuild-kinds-tags": "python scripts/rebuild-kinds-tags.py"
+    "rebuild-kinds-tags": "python scripts/rebuild-kinds-tags.py",
+    "sync-api-docs": "/bin/sh scripts/build-api-docs.sh",
+    "build-api-docs": "/bin/sh scripts/build-api-docs.sh"
```

Comment:
Only kept sync-api-docs to prevent Vercel build from breaking temporarily. Eventually can be removed once this is in _master_.